statement.md

```diff
@@ -0,0 +1,116 @@
+
+* There is an acute shortage of cybersecurity professionals and qualified software engineers. Moreover, universities are largely moving away from memory-unsafe languages like C in their undergraduate courses. 
+
+All of this demands a shift in mindset: from reactive patching to proactive prevention.
+
+Considering security earlier in the development lifecycle is not a new concept, but certainly no longer optional. This has been recognized in Europe by the passing of the Cyber Resilience Act (CRA) and its security-by-design requirement. 
+
+As the standards for the CRA are currently being developed, it’s important to consider memory safety early in their design, and include strategies to shift towards memory-safe tech where feasible, and include resources for remediation where not.
+
+# Europe Has the Groundwork, but needs to act decisively
+
+For decades, memory unsafe languages like C and C++ were the only viable option for performance-critical systems such as operating systems, browsers, and networking stacks. This is no longer the case.
+
+With the rise of modern, performant memory safe languages—most notably Rust—it is now feasible to build even low-level systems securely. Furthermore, we are seeing encouraging advances towards memory-safe C++. 
+
+That said, we are not advocating a wholesale rewrite of legacy systems. That would not be feasible. But we can and should:
+
+* Use memory safe languages for the development of new code or components; 
+
+* Gradually transition critical components in legacy systems where feasible;
+
+* Introduce meaningful mitigations to reduce the risk in existing and legacy memory-unsafe systems. 
+
+The path towards memory safety is not an easy one, but in our view the long-term security, economic, and societal benefits far outweigh the costs of inaction. Europe already has a strong foundation for this transition:
+
+* Developers and companies in Europe have been significant contributors to open-source, memory safe critical infrastructure, such as implementations of the critical internet protocols Domain Name System (DNS) and Network Time Protocol (NTP), as well as in industries such as energy and automobile sectors;    
+* There is growing adoption of memory safe languages in critical infrastructure applications and industry, according to the latest State of Rust survey. 
+
+While the foundation exists, to get to a memory-safe future, we need shifts in development teams and supply chains, as well as long-term planning and coordinated action across industries and governments.
+
+# We Call Upon Government and Industry to:
+
+* **Take a decisive stance**: Formally recognize memory safety as a cornerstone of Europe's cyber resilience strategy and include specific memory safety requirements in the implementation guidelines for the Cyber Resilience Act. Where existing memory-unsafe languages are in use, this would include recommendations on measures to reduce risk where feasible.
```

Comment:
"What specific guidelines are you targeting with this statement? Is this comment addressed to the standardisation efforts currently underway at CEN/CENELEC/ETSI? Or someone/something else?"
See #17